css/mainStyle.css

```diff
@@ -0,0 +1,204 @@
+
+.main-container{
+	display: flex;
+	flex-direction:row;
+	flex-wrap:wrap;
+	width: 100%;	
+	border: 1px solid gray;
+	margin: auto;
+}
+
+.side-bar{
+	width: 252px;
+	background-color: white;
+}
+
+.search-box{
+	display: flex;
+    /* flex-direction: row; */
+    justify-content: center;
+    height: 32px;
+    width: 240px;
+    background-color: #a0a0a0;
+    /* padding: 4 0 4 0px; */
+    /* margin: 2 2 2 2px; */
+}
+
+.input-search-box{
+	height:20px;
+	margin-top: 5px;
+}
+
+.btn-search-box{
+	background-color: #d83c3c;
+	width: 56px;
+	height:20px;
+	margin-top: 5px;
+	border: none;
+}
+
+.dashboard{
+	display: flex;
+	flex-direction:column;
+	align-items:center;
+	height: 575px;
+	width: 240px;
+	background-color: #a0a0a0;
+	    padding: 0px 0 0px 0px;
```

Comment:
padding: 0 will work the same, avoid writing extra characters/lines, and please keep indentation in mind.